echidna/cards/channels.py

```diff
@@ -61,6 +66,26 @@ def __init__(self, name):
         except ValueError:
             pass
 
+    def subscribe(self, client, last_seen=None):
+        super(RedisChannel, self).subscribe(client, last_seen)
+        # record uuid in redis
+        now = datetime.datetime.now()
+        bucket = "%s-%s" % (self.name, now.strftime("%Y%m%d%H"))
+        client_id = getattr(client, "given_id", "Anon")
+        if client_id != "Anon":
+            self._redis.sadd(bucket, client_id)
```

Comment:
I would recommend using hyperloglog for this (Redis commands ``PFADD`` and ``PFCOUNT``) rather than an ever-growing set.